src/main/java/com/ibm/crypto/plus/provider/XDHPublicKeyImpl.java

```diff
@@ -175,7 +175,7 @@ public XDHPublicKeyImpl(OpenJCEPlusProvider provider, AlgorithmParameterSpec par
 
         try {
             if (u == null) {
-                int pub_size = CurveUtil.getPublicCurveSize(curve);
+                int pub_size = CurveUtil.getCurveSize(curve);
```

Comment:
Should we change the name of the variable to `keySize`?

Reply:
Yes.